src/components/toggle/toggle.svelte

```diff
@@ -150,6 +143,18 @@
     -webkit-tap-highlight-color: transparent;
   }
 
+  @media (prefers-reduced-motion) {
```

Comment:
note: modern CSS doesn't support nesting media queries so we have to unnest them